_posts/2022-11-25-dgtal-release1.3.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: DGtal release 1.3
+---
+
+We are thrilled to announce the [release  1.3 of DGtal](https://github.com/DGtal-team/DGtal/releases/tag/1.3) and its tools. Many edits and bugfixes are listed in the [Changelog](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md), and we would like to thank all devs involved in this release. In this short review, we would like to focus on only few of them.
+
+
+* **Geometry processing**: many new features for the geometry processing of digial objects and surfaces
+  * a new [differential calculus framework](https://dgtal-team.github.io/doc-nightly/modulePolygonalCalculus.html) for polygonal meshes and digital surfaces (scalar field and vector field processing on surfaces)
+
+| | |
+|---|---|
+| ![](../img/polydec.png)| ![](../img/vf.png)|
+
+  * a new digital [convexity module](https://dgtal.org/doc/stable/moduleDigitalConvexity.html) allowing efficient convexity tests, with application to shape analysis and modelling.
```

Comment:
Module digital convexity is not new (since 1.1), but it has been improved to work in arbitrary dimension, and to count faster lattice points within polytope. moduleEnvelope is new. For the images, you can pick perhaps the lion digital polyhedra:
DGtal/geometry/doc/images/lion-tri-h0_5-sstd-all.jpg
DGtal/geometry/doc/images/lion-tri-h0_5-sstd-edges.jpg
or the spot examples:
DGtal/geometry/doc/images/spot-h0_005-all.jpg
DGtal/geometry/doc/images/spot-h0_005-edges.jpg